pkg/apis/componentconfig/types.go

```diff
@@ -684,9 +684,11 @@ type KubeControllerManagerConfiguration struct {
 	ConcurrentSATokenSyncs int32
 	// lookupCacheSizeForRC is the size of lookup cache for replication controllers.
 	// Larger number = more responsive replica management, but more MEM load.
+	// DEPRECATED: This is no longer used.
```

Comment:
It seems like the struct for kube-controller-manager doesn't exist in `componentconfig/v1alpha1`.

I added the timeline and linked to the GitHub tracking issue.